### ISSUE TITLE
Accessible Drag and Drop

### ISSUE THREAD

**Issue by ajrbyers** [BirkbeckCTP/janeway#3801]
**Is your feature request related to a problem? Please describe.**
The current implementation of drag and drop using JQUI Sortable is not accessible, namely it cannot be controlled with keyboard and mouse only and is not screen reader friendly.

**Describe the solution you'd like**
Re-implement sorting using accessible methods.

**Additional context**
W3C advice: https://www.w3.org/wiki/PF/ARIA/BestPractices/DragDrop
Interesting article by Salesforce: https://medium.com/salesforce-ux/4-major-patterns-for-accessible-drag-and-drop-1d43f64ebf09
